src/gentropy/locus_breaker_clumping.py

```diff
@@ -0,0 +1,75 @@
+"""Step to apply linkageg based clumping on study-locus dataset."""
+
+from __future__ import annotations
+
+from gentropy.common.session import Session
+from gentropy.dataset.summary_statistics import SummaryStatistics
+from gentropy.method.locus_breaker_clumping import LocusBreakerClumping
+
+
+class LocusBreakerClumpingStep:
+    """Step to perform locus-breaker clumping on a study."""
+
+    def __init__(
+        self,
+        session: Session,
+        summary_statistics_input_path: str,
+        clumped_study_locus_output_path: str,
+        lbc_baseline_pvalue: float,
+        lbc_distance_cutoff: int,
+        lbc_pvalue_threshold: float,
+        lbc_flanking_distance: int,
+        large_loci_size: int,
+        wbc_clump_distance: int,
+        wbc_pvalue_threshold: float,
+        collect_locus_distance: int,
+        collect_locus: bool = False,
+        remove_mhc: bool = True,
+    ) -> None:
+        """Run locus-breaker clumping step.
+
+        This step will perform locus-breaker clumping on the full set of summary statistics.
+        StudyLocus larger than the large_loci_size, by distance, will be further clumped with window-based
+        clumping.
+
+        Args:
+            session (Session): Session object.
+            summary_statistics_input_path (str): Path to the input study locus.
+            clumped_study_locus_output_path (str): path of the resulting, clumped study-locus dataset.
+            lbc_baseline_pvalue (float): Baseline p-value for locus breaker clumping.
+            lbc_distance_cutoff (int): Distance cutoff for locus breaker clumping.
+            lbc_pvalue_threshold (float): P-value threshold for locus breaker clumping.
+            lbc_flanking_distance (int): Flanking distance for locus breaker clumping.
+            large_loci_size (int): Threshold distance to define large loci for window-based clumping.
+            wbc_clump_distance (int): Clump distance for window breaker clumping.
+            wbc_pvalue_threshold (float): P-value threshold for window breaker clumping.
+            collect_locus_distance (int): Distance to collect locus.
+            collect_locus (bool, optional): Whether to collect locus. Defaults to False.
+            remove_mhc (bool, optional): If true will use exclude_region() to remove the MHC region.
+        """
+        sum_stats = SummaryStatistics.from_parquet(
+            session, summary_statistics_input_path
+        )
+        lbc = sum_stats.locus_breaker_clumping(
+            lbc_baseline_pvalue,
+            lbc_distance_cutoff,
+            lbc_pvalue_threshold,
+            lbc_flanking_distance,
+        )
+        clumped_result = LocusBreakerClumping._process_locus_breaker(
```

Comment:
Methods with underscore are conventionally not accessed publicly (in this sense gentropy steps are considered outside usage of `LocusBreakerClumping` class). This is how we use them: 

```python
# Calling public method:
object.public_method()

# within the public_method there's an internall call:
self._internal_method()
```

So there are two options I would choose from after considering if it really makes sense to use `locus_breaker_clumping()` output without post-processing.
1. If yes, then rename `_process_locus_breaker` to `process_locus_breaker`
2. If not, I would internalise  `_process_locus_breaker` I would call this function _inside_ `locus_breaker_clumping()`

Either way, I would rename this function to give a more informative name. What does "processing" mean in this context?

Reply:
This makes sense regarding underscores. 
 
Would `process_locus_breaker_output` be more informative?

Reply:
I would rather call it something like "resolve_long_loci" (ironically this is the function, that actually breaks long loci, not the first one, as that only defines the loci)

Reply:
However, that function not only resolves the long loci, but merges with the short loci, so It's fine to call `process_locus_breaker_output`.